.changeset/forty-badgers-clap.md

```diff
@@ -0,0 +1,7 @@
+---
+"@skeletonlabs/skeleton-svelte": patch
+"@skeletonlabs/skeleton-react": patch
+---
+
+chore: use shared classes
```

Comment:
Two suggestions:

- Probably worth of calling this a feature, so move up to `minor`
- Let's make the description a bit more verbose

"Implements the new common style system for all framework components"